services/ux-backend/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+var tokenLifetimeInHours int
+var listenPort int
+
+func init() {
```

Comment:
I didn't mean to push all logic to init, I suggested only for lifetime and that too in handler file. init should be used sparingly.